index.js

```diff
@@ -38,6 +38,10 @@ var validate = module.exports = function (name) {
     errors.push('name cannot start with an underscore')
   }
 
+  if (name.length && !name.match(/^[a-z0-9@]/)) {
+    warnings.push('name can only start with lowercase character and digit')
```

Comment:
Because this condition is only valid for new packages. If we convert this into an error, then we will get error while publishing older packages that do not support this condition.

Reply:
won't it be explicit in line 69 (72)? why this change is needed?

Reply:
Initially, we allowed the special characters (except dot and underscore)  as a first letter for the package name. e.g. there is a package name `-` exists. 
Now, the package name should only start with the lowercase character and the digit. 

Overall the validation happens in two places. one is on `user-acl-two` side and other is in this package.